xwiki-platform-core/xwiki-platform-mentions/xwiki-platform-mentions-ui/src/main/resources/XWiki/Mentions/MentionsMacro.xml

```diff
@@ -375,11 +375,14 @@ require(['deferred!ckeditor', 'xwiki-suggestUsers', 'jquery', 'xwiki-meta'], fun
       <code>.xwiki-mention {
   background-color: $services.mentions.mentionsColor;
   border-radius: 10px;
-  padding: 2px 5px 2px 5px;
+  padding: 1px 5px 1px 5px;
+  border: 1px solid @dropdown-divider-bg;
 }
 
 .xwiki-mention.user.self {
   background-color: $services.mentions.selfMentionsColor;
+  color: @btn-danger-color;
```

Comment:
It doesn't feel good to use that here: what if an admin configures the mention to use the same color as `@btn-danger-color` for background? Feels like this color should also be configurable...

Reply:
> what if an admin configures the mention to use the same color as @btn-danger-color for background? 

That's a mistake, similar to setting text in white in Iceberg, we can't do much to prevent that kind of error... As of now `*-color` are for foreground colors, and `*-bg` are for background colors. The first one is not really obvious.

The naming convention in this module are not making things easier... they use a `*Color` structure for the parameter names for backgrounds.

I don't know why only the background was made configurable :thinking: 
Updated those color parameters in https://github.com/xwiki/xwiki-platform/pull/2638/commits/ae70d914396f28a86088b248d6ce056878d8abb4 :+1: